django_mercadopago/models.py

```diff
@@ -189,6 +193,10 @@ class Preference(models.Model):
         max_digits=15,
         decimal_places=2,
     )
+    quantity = models.IntegerField(
```

Comment:
Probably best to restrict this to positive integers.

```suggestion
    quantity = models.`PositiveIntegerField`(
```